### ISSUE TITLE
Less disruptive UI for create meaning / reading note entry point

### ISSUE THREAD

**Issue by aaron-meyers** [davidsansome/tsurukame#511]
The recently added "Create meaning note" and "Create reading note" items are very distracting:

![image](https://user-images.githubusercontent.com/13340718/130512800-c563a0f1-7848-46c1-a02d-176b46854559.jpeg)

It looks like another meaning or reading and makes it harder to just read the existing meaning / reading content.

Suggestions:
- Right- or center-align the "Create...note" text 
- Use a + icon instead of text (or maybe a right-aligned **+note**)

**Comment by UInt2048** [comment]
See discussion at #486

**Comment by aaron-meyers** [comment]
Yeah I've seen the option for disabling it during reviews - in my case, it's mostly doing lessons where I find it distracting. I'd rather not turn it off since I occasionally add notes but it would be nice to at least make it center or right aligned to differentiate it.

**Comment by aaron-meyers** [comment]
Another thought on this - what if the **Create note** options were simply moved to the **Meaning explanation** and **Reading explanation** sections? This is more in line with how they are presented in WaniKani, as user-provided notes to help with memorization rather than alternate meanings / readings (there is already a feature for alternate meanings - user synonyms - and alternate readings don't really make sense). It could still be nice to make the UI more subtle per the suggestions above but moving it to under the explanations would mitigate this a lot (and seems more correct anyway). 

Thoughts?

**Comment by UInt2048** [comment]
@aaron-meyers That's actually a great idea!
